tests/page_tests.py

```diff
@@ -97,17 +101,20 @@ def remove_bold_and_italics(text):
                     )
                 )
             )
+        ## translate string to lsit of words
+        list_of_words = re.findall(r"\w+", text)
+        # check if list of words exist in the dictionary
+        errors = checker_py.unknown(list_of_words)
 
-        filters = [URLFilter, EmailFilter, MentionFilter, WikiWordFilter]
```

Comment:
Getting rid of all these filters means we're now gonna have loads more errors for emails and urls, which we needn't check.